### PR TITLE
Rename AlphaStack to VerticalStack

### LINKED ISSUE #8878: [Layout foundations] Rename `AlphaStack` to `VerticalStack`

### PR DESCRIPTION
**Pull request by alex-page**
### WHY are these changes introduced?

With the UI consolidation effort happening we are aligning component names and properties across repositories. This is a first step to align component names.

### WHAT is this pull request doing?

**⚠️ Breaking change** to rename `AlphaStack` to `VerticalStack`

### CODE REVIEW

File: polaris.shopify.com/content/components/layout-and-structure/vertical-stack.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Alpha stack
+title: VerticalStack
```
Comment (laurkim) [inline polaris.shopify.com/content/components/layout-and-structure/vertical-stack.md]:
```suggestion
title: Vertical stack
```

File: .changeset/silly-meals-hang.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/polaris': minor
+'polaris.shopify.com': minor
+---
+
+Rename AlphaStack to VerticalStack
```
Comment (laurkim) [inline .changeset/silly-meals-hang.md]:
```suggestion
Renamed AlphaStack to VerticalStack
```

**Review by laurkim — APPROVED** [review]

File: .changeset/silly-meals-hang.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/polaris': minor
+'polaris.shopify.com': minor
+---
+
+Renamed AlphaStack to VerticalStack
```
Comment (chloerice) [inline .changeset/silly-meals-hang.md]:
```suggestion
Renamed `AlphaStack` to `VerticalStack`
```

File: polaris.shopify.com/CHANGELOG.md [hunk 4]
```diff
@@ -149,7 +149,7 @@
 
 * [#8700](https://github.com/Shopify/polaris/pull/8700) [`47b03e639`](https://github.com/Shopify/polaris/commit/47b03e63956c3fd5a6e9ae479256c5b49a25882e) Thanks [@kyledurand](https://github.com/kyledurand)! - Added inheritance examples to Text component
 
-- [#8676](https://github.com/Shopify/polaris/pull/8676) [`b4a1c451f`](https://github.com/Shopify/polaris/commit/b4a1c451f9f911f81cd8c3d34b3a07743e72a346) Thanks [@laurkim](https://github.com/laurkim)! - Updated `AlphaStack` to support horizontal alignment with `inlineAlign` and vertical alignment with `align`
+- [#8676](https://github.com/Shopify/polaris/pull/8676) [`b4a1c451f`](https://github.com/Shopify/polaris/commit/b4a1c451f9f911f81cd8c3d34b3a07743e72a346) Thanks [@laurkim](https://github.com/laurkim)! - Updated `VerticalStack` to support horizontal alignment with `inlineAlign` and vertical alignment with `align`
```
Comment (chloerice) [inline polaris.shopify.com/CHANGELOG.md]:
```suggestion
- [#8676](https://github.com/Shopify/polaris/pull/8676) [`b4a1c451f`](https://github.com/Shopify/polaris/commit/b4a1c451f9f911f81cd8c3d34b3a07743e72a346) Thanks [@laurkim](https://github.com/laurkim)! - Updated `AlphaStack` to support horizontal alignment with `inlineAlign` and vertical alignment with `align`
```

Should we leave these old change log entries as AlphaStack?

**Review by chloerice — APPROVED** [review]

File: polaris.shopify.com/CHANGELOG.md (hunk 4 above)
Comment (laurkim) [reply polaris.shopify.com/CHANGELOG.md]: Pushed up a [commit](https://github.com/Shopify/polaris/pull/8935/commits/be32fe0a2379b86ecc05ada01f37249c32c480f4) to undo changes to changelog

**Pull request merged**